src/View/AdminAdmission/index.html.twig

```diff
@@ -13,7 +13,7 @@
             </thead>
             <tbody>
             {% for folder in folders %}
```

Comment:
si je ne dis pas de betise, en twig pour gagner un peu de temps, tu peux faire direct un 
`{% for folder in folders if folder.status is same as('En attente') %}`
(avec bien sur status et pas statut, voir remarque plus haut :p)